assets/wizards/advertising/views/ad-unit/index.js

```diff
@@ -104,15 +108,33 @@ class AdUnit extends Component {
 					</Card>
 				) ) }
 				<Button
-					isLink
+					isSecondary
 					onClick={ () => this.handleOnChange( 'sizes', [ ...sizes, adUnitSizes[ 0 ] ] ) }
 				>
-					{ __( 'Add Size', 'newspack' ) }
+					{ __( 'Add new size', 'newspack' ) }
 				</Button>
 				<div className="clear" />
+				<CheckboxControl
+					label={ __( 'Fluid size for native ads', 'newspack' ) }
+					onChange={ value => this.handleOnChange( 'fluid', value ) }
+					checked={ fluid }
+					help={ __(
+						'Fluid is a native ad size that allows more flexibility when styling your ad. Ad Manager automatically sizes the ad by filling the width of the enclosing column and adjusting the height as appropriate (just like a regular HTML div on your site).',
```

Comment:
Mentioning HTML might be too technical :) I think it's understandable without it.

```suggestion
						'Fluid is a native ad size that allows more flexibility when styling your ad. Ad Manager automatically sizes the ad by filling the width of the enclosing column and adjusting the height as appropriate.',
```